src/App.tsx

```diff
@@ -1,9 +1,10 @@
+import { WeatherBackground } from 'components/WeatherBackground';
 import styles from './App.module.scss';
 
 export const App = () => {
   return (
     <div className={styles.app}>
-      <h1 className={styles.appTitle}>Weazee</h1>
+      <WeatherBackground type="Sun" />
```

Comment:
It is looks weird that we use some component to set global background of the app